08_voting/index.tex

```diff
@@ -1 +1,192 @@
+
+\textbf{Borda Count} method will allow voters to rank preference instead of choosing one option only. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all voters' ballots produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the collective decision.
+
+% The tied results are resolved by selecting the most first-nominated candidate. If this still gives multiple winners, the most second-nominated candidate will be selected as the winner. This iterative counting process will act as a tiebreaker for winner selection.
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which is used to pick a single winner from a list of several candidates. 
+%Normally, voters cast a single vote for their preferred candidate, but in our game specification it is also possible for voters to make their preference list.
+Runoff voting is widely recognized and used around the world primarily for the election of the head of state. It is an effective voting method for the selection of candidates.
+
+\texttt{Voter} islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, two candidates with most first placed votes are chosen among all candidates. The Runoff voting can conclude the election in the first-round if one candidate obtains a majority. Otherwise, the second round need to be considered and utilized where the ballots will be recounted by eliminating the candidates other than the top-two. Subsequently, the candidate with the most votes between the top-two candidates will be the winner.
+
+\textbf{Instant Runoff} is a voting method which is used in single-seat elections which consist of more than two candidates. 
+Individual voters rank their preference of candidates. The candidate who gets the least number of first-placed is eliminated and this step is repeated until only one candidate remains.
+
+% In the event of tied result, alternative rules can be chosen to give a single winner. It is obvious that the effectiveness of this method comes from its flexibility of various rules available in a tie situation.
+
+\textbf{Approval} is a voting method for single winner in which each voter selects a subset of candidates. As of its procedures and usage in the game, Approval will be used for the elections of roles where each voter picks a group of candidates and the most named or approved candidate wins.
```

Comment:
suggested re-writing:

DELETE(As of its procedures and usage in the game). let's not say ''game'' but coursework. 

Approval will be used for the elections of roles where each voter picks a group of candidates**,** and **the most named candidate wins.**

do not forget ","

Reply:
changed